src/azure-cli/azure/cli/command_modules/storage/_exception_handler.py

```diff
@@ -0,0 +1,12 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+def file_related_exception_handler(ex):
+    from azure.cli.core.azclierror import InvalidArgumentValueError
+    if isinstance(ex, (FileNotFoundError, PermissionError, IsADirectoryError, OSError)):
+        raise InvalidArgumentValueError(ex)
```

Comment:
@bim-msft has defined a new error type `FileNotFoundError`, seems better to use this one.

#15779

Reply:
> @bim-msft has defined a new error type `FileNotFoundError`, seems better to use this one.
> 
> #15779

Sounds good. Also added other os error types